roles-training.md

```diff
@@ -0,0 +1,28 @@
+This document defines the training needed for each of the roles involved in [competency change].
```

Comment:
Do CTO and Driver not need training?

Even if they don't, I would still list that here.

Reply:
Good point!  Added.

Reply:
Perfect